libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -213,9 +270,19 @@ void AP_Frsky_Telem::send_SPort_Passthrough(void)
     for (int16_t i = 0; i < numc; i++) {
         prev_byte = _passthrough.new_byte;
         _passthrough.new_byte = _port->read();
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+        if (parse_sport_telemetry_data(_passthrough.new_byte, _SPort_bidir.parse_state)) {
+            process_sport_telemetry_packet(_SPort_bidir.parse_state.telemetry_rx_buffer);
+        }
+#endif //HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
     }
+
     if (prev_byte == FRAME_HEAD) {
-        if (_passthrough.new_byte == SENSOR_ID_27) { // byte 0x7E is the header of each poll request
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+        if (_passthrough.new_byte == SENSOR_ID_27 || _passthrough.new_byte == _SPort_bidir.downlink1_sensor_id || _passthrough.new_byte == _SPort_bidir.downlink2_sensor_id ) { // byte 0x7E is the header of each poll request
```

Comment:
Perhaps create an `is_passthrough_byte(...)` method to do this and tidy up here a bit.